scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -125,15 +125,28 @@ class MicrobeEditor{
             playerSpecies.organelles);
 
         editedMicrobe.resize(0);
+        playerSpecies.stringCode="";
         for(uint i = 0; i < templateOrganelles.length(); ++i){
             editedMicrobe.insertLast(cast<PlacedOrganelle>(templateOrganelles[i]));
+            playerSpecies.stringCode+=cast<PlacedOrganelle>(templateOrganelles[i])._organelle.gene;
```

Comment:
The cast here should only be done once as it is (very) expensive.

Reply:
Okay, will fix that

Reply:
fixed